### ISSUE TITLE
Implement new login screen for Janus.

### ISSUE THREAD

**Issue by myleshorton** [getlantern/lantern#2223]

**Comment by oxtoacart** [comment]
We'll have to define how we want the authentication process to work.  Depending on what we do, there may or may not be an actual "login" screen, but there will almost certainly be some screen.

**Comment by ed2k** [comment]
why we need a screen? for normal get mode users, the best case is, it just works out of the box (maybe with an animated tray icon). the gui could be for advanced user or trouble shooting.

**Comment by oxtoacart** [comment]
> why we need a screen? for normal get mode users, the best case is, it just works out of the box (maybe with an animated tray icon). the gui could be for advanced user or trouble shooting.

I agree.  The main purpose of authentication is to provide a foundation for P2P services.  Lantern can and should work out-of-the-box without the user having to use a GUI.  However, if and when the user is ready to engage in P2P Lantern, there will be some sort of authentication required.

There are plenty of users who have already shown that their interested in P2P by adding friends and I believe that they will choose to log in if given the option.  I think the key change from today is just be to make logging in optional rather than required.

Note - this is just my opinion and doesn't represent official production direction.  This is all still very much under discussion.
